domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/YesNoOption.java

```diff
@@ -11,6 +12,10 @@
     NO;
 
     public static YesNoOption fromValue(String value) {
+        if (StringUtils.isBlank(value)) {
```

Comment:
From value should throw exception if the value is not mappable. null check should be in another method like ofNullable()